spec/models/gids_redis_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'common/exceptions'
+
+describe GIDSRedis do
+  subject { GIDSRedis.new }
+
+  let(:scrubbed_params) { {} }
+  let(:body) { {} }
+  let(:gids_response) do
+    GI::GIDSResponse.new(status: 200, body: body)
+  end
+
+  describe 'method_missing' do
+    it 'returns response body' do
+      allow_any_instance_of(GI::Client).to receive(:get_institution_details).and_return(gids_response)
+
+      expect(subject.get_institution_details(scrubbed_params)).to eq(gids_response.body)
+    end
+
+    it 'undefined method' do
+      expect{ subject.not_a_real_method }.to raise_error(NoMethodError)
+    end
+  end
+
+  describe 'cached attributes' do
+    context 'get_calculator_constants' do
```

Comment:
I don't think this line is necessary.  We are using `get_calculator_constants` but these specs are demonstrating caching of any method.

Reply:
removed